narwhals/_duckdb/expr.py

```diff
@@ -63,6 +65,7 @@ class DuckDBExpr(LazyExpr["DuckDBLazyFrame", "duckdb.Expression"]):
     def __init__(
         self,
         call: EvalSeries[DuckDBLazyFrame, duckdb.Expression],
+        previous_call: EvalSeries[DuckDBLazyFrame, duckdb.Expression] | None = None,
```

Comment:
does this need to be an argument to `DuckDBExpr`? I think it can be removed, as it's set in `_with_unorderable_window_function`